src/app/core/guards/onboarding.guard.ts

```
@@ -8,18 +8,23 @@
   providedIn: 'root',
 })
 export class OnboardingGuard implements CanActivate {
-  constructor(private spenderOnboardingService: SpenderOnboardingService) {}
+  constructor(private spenderOnboardingService: SpenderOnboardingService, private router: Router) {}
 
   canActivate(
-    next: ActivatedRouteSnapshot,
-    state: RouterStateSnapshot
+    _: ActivatedRouteSnapshot,
+    _: RouterStateSnapshot
```

Biome (1.9.4)

[error] Duplicate parameter name.

The parameter overrides a preceding parameter by using the same name.

(lint/suspicious/noDuplicateParameters)